openapi3.yaml

```diff
@@ -1,45 +1,124 @@
 openapi: 3.0.1
 info:
-  title: service-name
-  description: basic template for map colonies service
+  title: mapproxy-api
+  description: rest api service to manage layers in mapproxy
   version: 1.0.0
   license:
     name: MIT
     url: https://opensource.org/licenses/MIT
 servers:
   - url: http://localhost:8080
 paths:
-  /resourceName:
+  /layer:
     get:
-      operationId: getResourceName
+      operationId: getLayer
       tags:
-        - resourceName
-      summary: gets the resource
+        - Layer
+      summary: get exiting layer
       responses:
         200:
           description: OK
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/resource'
+                $ref: '#/components/schemas/layer'
+        500:
+          description: Ineternal Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/error'
+    post:
+      operationId: addLayer
+      tags:
+        - Layer
+      summary: add layer to mapproxy
+      responses:
+        201:
+          description: created
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/layer'
+        400:
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/error'
+        500:
+          description: Ineternal Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/error'
+  /best:
+    get:
+      operationId: getBest
+      tags:
+        - Best
+      summary: get best layers
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/best'
+        500:
+          description: Ineternal Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/error'
     post:
-      operationId: createResource
+      operationId: addBestLayer
```

Comment:
addBestLayer as the url will include best shouldn't we call it just addLayer
"/best/addLayer"

Reply:
fixed in 924cce1